File: dotnet/source/amp.eventing/streaming/IStandardStreamingEventBus.cs
using amp.messaging;
using cmf.bus;
using cmf.eventing.patterns.streaming;

namespace amp.eventing.streaming
{
    public interface IStandardStreamingEventBus : IStreamingEventBus , IMessageProcessor
    {
        IEnvelopeBus EnvelopeBus { get; }
       
    }
}

File: java/amp-eventing/src/main/java/amp/eventing/streaming/IStandardStreamingEventBus.java
package amp.eventing.streaming;

import amp.messaging.IMessageProcessor;
import cmf.bus.IEnvelopeBus;
import cmf.eventing.patterns.streaming.IStreamingEventBus;

public interface IStandardStreamingEventBus extends IStreamingEventBus, IMessageProcessor {
   
	IEnvelopeBus getEnvelopeBus();
}
